lib/LoginStrategy.ts

```diff
@@ -239,4 +253,49 @@ export class LoginStrategy extends BaseStrategy {
       return acc
     }, [] as Claim[])
   }
-}
+
+  /**
+   * @description checks if the claimer is authenticated into the DID Document
+   *
+   * @param token
+   * @param payload
+   *
+   * @returns {string} issuer DID or null
+   */
+  async verifyDidClaim(token: string, payload: ITokenPayload): Promise<string> {
+
+    const didDocument = await this.didResolver.read(payload.iss)
```

Comment:
Better to use the `cacheServer` if it is available as it will potentially be much faster.
```suggestion
    const didDocument = await this.cacheServerClient?.getDidDocument(payload.iss) ??  await this.didResolver.read(payload.iss)
```